lib/counterfeit/railtie.rb

```diff
@@ -1,6 +1,7 @@
 module Counterfeit
   class Railtie < Rails::Railtie
+    initializer 'counterfeit_railtie.logger' do
+      Counterfeit.logger = Rails.logger
+    end
   end
 end
-
-require 'counterfeit/railtie' if defined?(Rails)
```

Comment:
This is suppsoed to be in the main included lib file and not in the railtie directory.